webapp/src/main/webapp/WEB-INF/conf/news/news-configuration.xml

```diff
@@ -89,11 +89,13 @@
           <name>classpath.resources</name>
           <description>The resources that start with the following package name should be load from file system</description>
           <value>locale.portlet.news.News</value>
+          <value>locale.navigation.group.platform.web-contributors</value>
```

Comment:
not needed here IMHO, but we should add a value param init.resources with this value.
As I has mentionned in the last review, <description> to be deleted from classpath.resources and portal.resource.names values params since it is not relevant.